Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp

```diff
@@ -7766,6 +7788,7 @@ IGNORE_CLANG_WARNINGS_END
             ValueFromBlock notFoundResult = isArrayIncludes ? m_out.anchor(m_out.constBool(false)) : m_out.anchor(m_out.constIntPtr(-1));
             m_out.jump(continuation);
 
+            // TODO
```

Comment:
Is this `// TODO` comment a leftover? If yes please remove it. If not, can you comment more details?